src/CppAst.Tests/AttributesTest/TestTokenAttributes.cs

```diff
@@ -555,6 +555,95 @@ public void TestCppNoParseOptionsAttributes()
           );
         }
 
+        [Test]
+        public void TestCpp17UsingAttributes()
+        {
+            ParseAssert(@"
+struct Test{
+    int a;
+};
+using MyTest [[cppast(""old"")]] = Test;
+", compilation =>
+                {
+                    Assert.False(compilation.HasErrors);
+
+                    Assert.AreEqual(1, compilation.Typedefs.Count);
 
+                    Assert.AreEqual(1, compilation.Typedefs[0].TokenAttributes.Count);
+                    {
+                        var attr = compilation.Typedefs[0].TokenAttributes[0];
+                        Assert.AreEqual("cppast", attr.Name);
+                        Assert.AreEqual("\"old\"", attr.Arguments);
+                        Console.WriteLine("Done");
```

Comment:
Guess it can go

Reply:
Yes, it was left over debugging code. That was a challenge on this. I don't currently have a Windows system and for some reason which I still don't know why, each time I hit a breakpoint a SEH would occur. Luckily I could look at the stack and do everything else, I just could step or continue from a breakpoint. So I had to use a combination of WriteLine statements, single use breakpoints and conditionals to track events. Primitive but it worked.